com.feaneron.Boatswain.json

```diff
@@ -0,0 +1,164 @@
+        "--device=all",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.mpris.MediaPlayer2.*"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        "shared-modules/libusb/libusb.json",
+        {
+            "name" : "hidapi",
+            "buildsystem" : "autotools",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/libusb/hidapi.git",
+                    "tag" : "hidapi-0.11.2",
+                    "x-checker-data" : {
+                        "type" : "git",
+                        "tag-pattern" : "^hidapi-([\\d.]+)$"
+                    }
+                }
+            ]
+        },
+        {
+            "name" : "gusb",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Ddocs=false",
+                "-Dtests=false",
+                "-Dvapi=false"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://people.freedesktop.org/~hughsient/releases/libgusb-0.3.10.tar.xz",
+                    "sha256" : "0eb0b9ab0f8bba0c59631c809c37b616ef34eb3c8e000b0b9b71cf11e4931bdc"
+                }
+            ]
+        },
+        {
+            "name" : "libpeas",
+            "buildsystem" : "meson",
+            "cleanup" : [
+                "/bin/*",
+                "/lib/peas-demo"
+            ],
+            "config-opts" : [
+                "-Dcatalogue=false",
+                "-Ddemos=false",
+                "-Dgtk_doc=false",
+                "-Dwidgetry=false"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://download.gnome.org/sources/libpeas/1.32/libpeas-1.32.0.tar.xz",
+                    "sha256" : "d625520fa02e8977029b246ae439bc218968965f1e82d612208b713f1dcc3d0e",
+                    "x-checker-data" : {
+                        "type" : "gnome",
+                        "name" : "libpeas"
+                    }
+                }
+            ]
+        },
+        {
+            "name" : "libportal",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Dbackends=gtk4",
+                "-Ddocs=false",
+                "-Dtests=false",
+                "-Dvapi=false"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/flatpak/libportal.git",
+                    "tag" : "0.6"
+                }
+            ]
+        },
+        {
+            "name" : "libadwaita",
```

Comment:
do you need more than what is in the GNOME 42 runtime?

Reply:
Yes, I need that because of AdwEntryRow and AdwPasswordEntryRow. I'll update the manifest with a stable libadwaita release when it's available, and will remove this module entirely with GNOME 43.

Reply:
For clarity, these widgets will be available with libadwaita 1.2. There is no release, not even unstable, with the relevant commits, which is why I pinned the manifest to a specific commit. As I said, I'll keep an eye on releases and update accordingly.

Reply:
The release of the alpha version of 1.2 will be around the time of the alpha release for GNOME 43 by the way, so this is the correct thing to do.